examples/tutorials/shared-todo-completed/pages/todos/[id].vue

```diff
@@ -147,8 +147,7 @@ async function addTodo() {
             protocolPath: 'list/todo',
             schema: protocolDefinition.types.todo.schema,
             dataFormat: protocolDefinition.types.todo.dataFormats[0],
-            parentId: listId.value,
-            contextId: listId.value,
+            parentContextId
```

Comment:
```suggestion
            parentContextId: listId.value
```

I think maybe you have a typo? Unless I'm wrong, but I checked out the branch and it was still broken forme.

Reply:
You are right. Typo on my end. Thanks for the catch. @blackgirlbytes